transfer-old-data/Dockerfile

```diff
@@ -5,7 +5,7 @@ COPY requirements.txt .
 RUN pip install -r requirements.txt
 RUN python -m pip install 'boto3-stubs[essential]'
 
-COPY ecs_load_to_s3.py .
+COPY transfer_old_data.py .
```

Comment:
Nice changes to the file name - makes it more easy to understand exactly what it's doing!